cmd/minikube/cmd/mount.go

```diff
@@ -191,6 +193,7 @@ var mountCmd = &cobra.Command{
 
 		err = cluster.Mount(co.CP.Runner, ip.String(), vmPath, cfg)
 		if err != nil {
+			node.ExitIfFatal(err)
```

Comment:
I dont think we should call the node package for this. node is meant for starting the node.

how about just checking the error type here and if that error type is do the right thing ?

Reply:
Done.